app/templates/home/show.hbs

```diff
@@ -1,28 +1,30 @@
 {{page-title model.title}}
 <div class="relative overflow-hidden show-poster">
   {{#if model.thumbImageUrl}}
-    <div style="background-image: url({{model.thumbImageUrl}})" alt={{model.title}} class="w-full h-full show-image"></div>
+  <div style="background-image: url({{model.thumbImageUrl}})" alt={{model.title}} class="w-full h-full show-image">
+  </div>
   {{else if model.host.imageMediumUrl}}
-    <div style="background-image: url({{model.host.imageMediumUrl}})" alt={{model.dj.username}} class="w-full h-full show-image"></div>
+  <div style="background-image: url({{model.host.imageMediumUrl}})" alt={{model.dj.username}}
+    class="w-full h-full show-image"></div>
   {{/if}}
   <div class="bg-df-pink show-info">
-    <h1 class="font-debussy text-3xl">
+    <h1 class="cursive text-3xl">
```

Comment:
Is the cursive class a tailwind built in one ?

Reply:
oh shoot it isn't. I think I have to add it to the config https://tailwindcss.com/docs/font-family#customizing

Reply:
@dudethatbe Add a `cursive` entry to the object here just like the others, and you should be able to use the `font-cursive` class everywhere.
https://github.com/datafruits/datafruits/blob/master/app/tailwind/config.js#L7